subsys/shell/modules/kernel_service.c

```diff
@@ -218,6 +222,66 @@ static int cmd_kernel_stacks(const struct shell *shell,
 }
 #endif
 
+static int cmd_kernel_sleep(const struct shell *sh,
+			    size_t argc, char **argv)
+{
+	ARG_UNUSED(sh);
+	ARG_UNUSED(argc);
+
+	k_msleep(strtoul(argv[1], NULL, 10));
+
+	return 0;
+}
+
+#if defined(CONFIG_LOG_RUNTIME_FILTERING)
+static int16_t log_source_id_get(const char *name)
+{
+	uint32_t count = log_src_cnt_get(CONFIG_LOG_DOMAIN_ID);
+	if (count > INT16_MAX) {
+		return -1;
+	}
```

Comment:
```suggestion
	uint32_t count = log_src_cnt_get(CONFIG_LOG_DOMAIN_ID);
	
	if (count > INT16_MAX) {
		return -1;
	}
```

Can you please add a comment for why we are checking if `count > INT16_MAX`? It's not really obvious :)